src/main/java/juja/sqlcmd/DataSet.java

```diff
@@ -19,6 +19,10 @@ public void insertValue(int columnIndex, String string) {
         row[columnIndex] = string;
     }
 
+    public String[] values() {
+        return row.clone();
```

Comment:
Why you use clone() instead of arrayCopy() or smth?

Reply:
arrayCopy() needs 5 parameters. 5, Carl!)